### PR TITLE
Add support for marking PR body content with an expiration time

### PR DESCRIPTION
**Pull request by rwestberg**
For information in the PR body that is calculated using data from external sources (like Jira, another personal fork, etc.) we need to perform a periodical refresh, as our normal way of detecting updates cannot be used (the updatedAt field of a pull request).

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Testing

|     | Linux x64 | Windows x64 | macOS x64 |
| --- | ----- | ----- | ----- |
| Build / test | ⏳ (1/1 running) | ⏳ (1/1 running) | ⏳ (1/1 running) |

### Reviewers
 * [Erik Helin](https://openjdk.java.net/census#ehelin) (@edvbld - **Reviewer**) ⚠️ Review applies to cd19ac11a56a9d040719a5194386545620d0c4af

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/862/head:pull/862`
`$ git checkout pull/862`

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java [hunk 1]
```diff
@@ -66,6 +66,11 @@ private String encodeReviewer(HostUser reviewer, CensusInstance censusInstance)
     String getMetadata(CensusInstance censusInstance, String title, String body, List<Comment> comments,
                        List<Review> reviews, Set<String> labels, boolean isDraft) {
         try {
+            var hasExpired = ExpirationTracker.hasExpired(body);
+            if (hasExpired) {
+                return Base64.getUrlEncoder().encodeToString(UUID.randomUUID().toString().getBytes(StandardCharsets.UTF_8));
```
Comment (edvbld) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java:71]:
```suggestion
                return String.valueOf(Math.random());
```

**Review by edvbld — APPROVED** [review]
Looks good, just a small suggestion to simplify returning a random string :smile:

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java (hunk 1 above)
Comment (rwestberg) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java]: Guess that makes sense. :)

**Pull request closed without merging**